<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>729</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>813</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="originalButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>580</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Origin</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagelabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>891</width>
      <height>650</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="meanfilterButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>120</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mediumfilterButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>190</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gaussianfilterButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>260</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="robertsoperatorButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>350</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Roberts Operator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prewittoperatorButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>420</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Prewitt Operator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sobeloperatorButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>490</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel Operator</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
